test/github-issues/4350/issue-4350.ts

```diff
@@ -0,0 +1,55 @@
+import "reflect-metadata";
+import {closeTestingConnections, createTestingConnections} from "../../utils/test-utils";
+import {Post1, TestEnum1} from "./entity/Post1";
+import {Post2, TestEnum2} from "./entity/Post2";
+import {Connection} from "../../../src";
+import {expect} from "chai";
+
+describe("github issues > #4350 Array of enums column doesn't work at all", () => {
+    let connections: Connection[];
+    after(() => closeTestingConnections(connections));
+
+    it("should migrate postgres enums correctly", async () => {
+        connections = await createTestingConnections({
+            entities: [__dirname + "/entity/Post1{.js,.ts}"],
+            schemaCreate: true,
+            dropSchema: true,
+            enabledDrivers: ["postgres"]
```

Comment:
Can this also be enabled for cockroach DB

Reply:
Well, I would just revert https://github.com/typeorm/typeorm/pull/6126/commits/9e2276c5b673f7c3d8ba7fab4bbc1d019a01f917 again. I just notice cockroachdb as a separate driver which doesn't include my fix. So the test will fail.

Reply:
The cockroachdb driver doesn't even handle enum cases. I think this is an issue by itself. I could open an issue for someone else.

Reply:
Ok, will remove it. Is not even listed here: https://typeorm.io/#/entities/column-types-for-cockroachdb